1-JavaScript/Week2/assignment/ex1-giveCompliment.js

```diff
@@ -1,30 +1,16 @@
 'use strict';
-/* -----------------------------------------------------------------------------
-Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-JavaScript/Week3#exercise-1-you-are-amazing
-
-1. Complete the function named `giveCompliment`as follows:
-
-   - It should take a single parameter: `name`.
-   - Its function body should include a variable that holds an array,
-     `compliments`, initialized with 10 strings. Each string should be a
-     compliment, like `"great"`, `"awesome"` and so on.
-   - It should randomly select a compliment from the array.
-   - It should return the string "You are `compliment`, `name`!", where
-     `compliment` is a randomly selected compliment and `name` is the name that
-     was passed as an argument to the function.
-
-2. Call the function three times, giving each function call the same argument:
-   your name.
-   Use `console.log` each time to display the return value of the
-   `giveCompliment` function to the console.
------------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
-}
+
+  function giveCompliment(name) {
+
+    const compliments = ["great","nice","amazing","very good","strong","perfect","very cool","fantastic","brilliant","awesome"];
```

Comment:
More stylistic, but I would add spaces between the elements of the array. If it becomes to long, I would also separate per line.

```suggestion
  function giveCompliment(name) {
    const compliments = ["great", "nice", "amazing", "very good", "strong", "perfect", "very cool", "fantastic", "brilliant", "awesome"];
```